src/Utility/CommandLine.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Utility;
+
+use Symfony\Component\Process\Process;
+
+class CommandLine
+{
+    /** @var string[] */
+    private array $args = [];
+
+    /**
+     * Constructor.
+     */
+    public function __construct(string|array $arg = [], bool $escape = true)
```

Comment:
as you use it only with a string as first argument, I think we should embrace KISS principle and only accept string here.

Reply:
I don't see any problem in accepting array as well, but I don't mind keeping only string either.

Reply:
I'd tend to keep it as-is and just close this thread. @SimonMellerin ?